cap/modules/theme/templates/cap_theme/header_login.html

```diff
@@ -25,9 +25,9 @@
 
 {% block navbar_login %}
   {%- if not current_user.is_authenticated %}
-  <form class="navbar-right">
-    <a  href=" {{url_for('invenio_oauthclient.login', remote_app='cern', next=url_for('cap_theme.index'))}}" class="btn btn-default sign-in-btn btn-lg" >{{_('Sign in with CERN')}}</a>
-  </form>
+  <div>
+    <a id="login-button"  href=" {{url_for('invenio_oauthclient.login', remote_app='cern', next=url_for('cap_theme.index'))}}" class="btn btn-primary btn-lg" >{{_('Log in with your CERN account')}}</a>
```

Comment:
Here remove ```id="login-button"``` and also from the homepage and create a class that will take care of the css.